test/Kestrel.Core.Tests/Http1ConnectionTests.cs

```diff
@@ -484,6 +484,12 @@ public async Task TakeStartLineThrowsOnNullCharInMethod(string method)
                 _http1Connection.TakeStartLine(readableBuffer, out _consumed, out _examined));
             _transport.Input.AdvanceTo(_consumed, _examined);
 
+            var index = requestLine.IndexOfAny(new char[] { '\r', '\n' });
```

Comment:
This one has an exact string too, use `$"{method} / HTTP/1.1" in assert.

Reply:
Ok.